flashlight/nn/modules/Conv2D.h

```diff
@@ -8,9 +8,11 @@
 
 #pragma once
 
+#include <memory.h>
```

Comment:
remove? I assume this is something internal :)

Reply:
Done.